### PR TITLE
Additional data to me and project_profile endpoints

### PR DESCRIPTION
**Pull request by gridcell**
- Social media image url
- number of active sample units

### CODE REVIEW

**Comment by gridcell** [comment]
[Updates to project profile endpoint](https://trello.com/c/yz9JbadA/308-updates-to-project-profile-endpoint)

File: src/api/resources/me.py [hunk 1]
```diff
@@ -19,6 +22,7 @@ class Meta:
             'email',
             'created_on',
             'updated_on',
+            'picture',
```
Comment (saanobhaai) [inline src/api/resources/me.py:25]: Since touching this file, do the Black/isort thing?

**Review by saanobhaai — CHANGES_REQUESTED** [review]
Just one potential formatting comment

File: src/api/resources/me.py (hunk 1 above)
Comment (gridcell) [reply src/api/resources/me.py:25]: Yeah I think I was seeing that in a few spots, was thinking after next api release we do https://trello.com/c/yHnMBnbM/826-add-black-code-formatting-on-pre-commit-hook

**Pull request merged**